src/sync/app.cpp

```diff
@@ -910,13 +909,13 @@ void App::call_function(std::shared_ptr<SyncUser> user,
     std::stringstream s;
     s << bson::Bson(args);
 
-    Request request {
-        .method = HttpMethod::post,
-        .url = route,
-        .body = s.str()
-    };
+    Request req;
+    req.method = HttpMethod::post;
+    req.url = route;
+    req.body = s.str();
```

Comment:
It looks like this change caused build failures in the app tests on the v10 branch. The previous way that the Request was initialized would zero initialize the omitted members, namely `uses_refresh_token` should be false here. The linux app tests did fail on this PR in CI. I can see that this error was obscured by a separate windows issue, but please be careful when merging PR's which fail CI.

Reply:
sorry about that